gradebook/app/ui/pom.xml

```diff
@@ -79,6 +79,10 @@
       <groupId>org.sakaiproject.edu-services.gradebook</groupId>
       <artifactId>gradebook-service-hibernate</artifactId>
     </dependency>
+    <dependency>
+        <groupId>org.sakaiproject.assignment</groupId>
+        <artifactId>sakai-assignment-api</artifactId>
+    </dependency>
    <!-- <dependency>
```

Comment:
I don't think there's anything wrong anymore with a tool depending on another tools api. Even in the past what the release process did was release all of the apis so depending on the api was the ideal thing to do. However in this case, I don't think gradebook specifically should have to depend on any tools api. There should be another fix for this, ideally one that allows assignment to correctly look up the gradebook entry even if the title is changed.